README.md

```diff
@@ -120,7 +123,15 @@ end
 ```
 
 ```rb
-# Your answer goes here...
+# $.ajax ({
```

Comment:
This question is not asking for an AJAX call. It's asking how you could set up the back end so that it serves up a list of posts when somebody sends a request asking for all the posts.